src/Components/Content.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+import { AreaChartOutlined } from '@ant-design/icons'
+import { Row, Col } from 'antd';
+
+type contentDetailsProps = {
+    id: any
+    title: any
+    subtitle: any
+    description: string
+}
+
+const Content = (props:contentDetailsProps) => {
+  return (
+    
```

Comment:
remove unnecessary spaces here! 
follow one convention, I think here it is 2 spaces everywhere

Reply:
just run format with preetier
i will set up a common .preetierrc soon

Reply:
I removed unwanted spaces.